packages/react-components/react-progress/src/stories/Progress/ProgressDefault.stories.tsx

```diff
@@ -2,13 +2,13 @@ import * as React from 'react';
 import { Progress, ProgressProps } from '@fluentui/react-progress';
 
 export const Default = (props: Partial<ProgressProps>) => {
-  return <Progress {...props} percentComplete={50} />;
+  return <Progress {...props} />;
 };
 
 Default.parameters = {
   docs: {
     description: {
-      story: `Default determinate Progress bar`,
+      story: `Default indeterminate Progress bar`,
```

Comment:
This is a bit of a style choice, but I'd actually recommend leaving the default story showing the determinate case (i.e. with `value={0.5}`), and have a separate story for the indeterminate case. Even though indeterminate is the "default" with no props, the determinate case seems like a more typical scenario.

Reply:
I follow that, I'll make the default `determinate` and change the `determinate` story to `indeterminate`